### DISCUSSION TITLE
Accepting cookie consent does not dismiss modal dialog

### DISCUSSION THREAD

**Discussion by joshooaj** [squidfunk/mkdocs-material#3215] [Q&A] [answered]
I'm using the anchor tag `#__consent` to enable the user to launch the cookie consent dialog on demand, and when you manually launch the dialog this way, the **Accept** button does not dismiss the dialog. It basically reloads the page with /#__consent still in the URL. So you have to manually remove the anchor tag from the URL to get it to go away.

Right now I have the link as a button in the footer. I put the link there using the copyright property in mkdocs.yml. Not the most obvious place for it but it was a quick and easy way to give the user access to it on any page. The behavior is the same though whether I do it this way, or put `[Change cookie settings](#__consent)` in the body of a page.

You can see the behavior on [milestonepstools.com](https://www.milestonepstools.com). Scroll down to the bottom of the page and click the "Change cookie settings" button.

**Comment by squidfunk** [comment, marked as answer]
Thanks for reporting. I've just created an issue out of your report #3216

**Reply by joshooaj** [reply]
Thanks!
